db/db_impl_compaction_flush.cc

```diff
@@ -706,7 +712,9 @@ Status DBImpl::CompactRange(const CompactRangeOptions& options,
     }
     s = RunManualCompaction(cfd, ColumnFamilyData::kCompactAllLevels,
                             final_output_level, options.target_path_id,
-                            options.max_subcompactions, begin, end, exclusive);
+                            options.max_subcompactions,
+                            options.bottommost_level_compaction, begin, end,
```

Comment:
We are passing three members of options in now. Do we consider to forward `const CompactRangeOptions&` instead?

Reply:
was thinking about that as well, will update the next upload